Tutorial.md

```diff
@@ -392,10 +392,10 @@ to_cldf(wl, path='cldf-new')
+Being able to export wordlists, with any additional information such as cognate sets coming from automatic cognate judgment, allows users to export, deploy, and share datasets easily. Both wordlists and CLDF datasets can be either provided as-is to users or, preferably, made available via on-line services such as GitHub and Zenodo. For CLICS, the Database of Cross-Linguistic Colexifications, we make heavy use of this and automatically publish all curated datasets from GitHub to the [CLICS Zenodo Community](https://zenodo.org/communities/clics/).
 
-An interesting possibility is to rely on the infrastucture of `lexibank`, to deploy datasets as Python packages that can then be conveniently installed from the command line, and [`clld`](https://clld.org), to deploy them as online services. This makes publishing and serving curated data straightforward, and also has the advantage of facilitating anonymous sharing of data for review purposes, deploying it as a Python package on GitHub and using it by means of on-line services such as the Open Science Framework.
+An interesting possibility is to rely on the infrastructure of `lexibank`, to deploy datasets as Python packages that can then be conveniently installed from the command line, and [`clld`](https://clld.org), to deploy them as online services. This makes publishing and serving curated data straightforward, and also has the advantage of facilitating anonymous sharing of data for review purposes, deploying it as a Python package on GitHub and using it by means of on-line services such as the Open Science Framework.
 
 ## 3 Conclusion
 
-Given the complexity of the topic, the current tutorial can only provide an incomplete introduction to the fascinating world of data managment in computational phylogenetics and computer-assisted language comparison. We hope, however, to have shown enough to ease the the entry for readers with little experience in data managment and phylogenetic analysis.  If you have questions on the management of linguistic data for phylogenetic analyses or CLDF, don't hesitate to contact us at <lexibank@shh.mpg.de>. Questions about LingPy can be posted either as an issue on its [GitHub repository](https://github.com/lingpy/lingpy) or sent to <info@lingpy.org>.
+Given the complexity of the topic, the current tutorial can only provide an incomplete introduction to the fascinating world of data management in computational phylogenetics and computer-assisted language comparison. We hope, however, to have shown enough to ease the entry for readers with little experience in data managment and phylogenetic analysis.  If you have questions on the management of linguistic data for phylogenetic analyses or CLDF, don't hesitate to contact us at <lexibank@shh.mpg.de>. Questions about LingPy can be posted either as an issue on its [GitHub repository](https://github.com/lingpy/lingpy) or sent to <info@lingpy.org>.
```

Comment:
"to the fascinating world of " -- a bit over the top :) Please delete.

Reply:
Oh, come on, will you tell me you're not thrilled about it, that kids don't say "when I grow up, I want to manage data"? (seriously, deleting :wink:)